streaming/src/test/scala/org/apache/gearpump/streaming/StreamingTestUtil.scala

```diff
@@ -37,4 +49,63 @@ object StreamingTestUtil {
     executorId += 1
     miniCluster.launchActor(props).asInstanceOf[TestActorRef[AppMaster]]
   }
+
+  def createEchoForTaskActor(taskClass: String, taskConf: UserConfig, system1: ActorSystem, system2: ActorSystem): (ActorRef, TestProbe) = {
+    import system1.dispatcher
```

Comment:
I will try to simplify this function. Two actor systems simulate the situation of TaskActor distributed on different Workers.